an_devices/advanced_navigation_device_serial.py

```diff
@@ -28,24 +28,19 @@
 ################################################################################
 
 import os
-import datetime
 import serial
 import serial.serialutil as serialutil
-from enum import Enum
-from struct import pack, unpack
-from dataclasses import dataclass, field
-from ctypes.wintypes import DOUBLE
-from anpp_packets.packets.an_packet_protocol import AN_Decoder
 from abc import ABC, abstractmethod
-from anpp_packets.packets.anpp_packets import ResetVerification, OdometerFlags, ExternalAirDataPacketFlags, PacketPeriod, GPIOOutputRate
 
-valid_baud_rates = [2400, 4800, 9600, 19200, 38400, 57600,
-                    115200, 230400, 250000, 460800, 500000,
-                    800000, 921600, 1000000, 1250000, 2000000]
+from anpp_packets import *
```

Comment:
It doesn't look like this is being used (+ generally good to avoid importing all)
```suggestion
```